### ISSUE TITLE
[discussion]What is the difference between these two pieces of code? And what is "fn":"Indepent()"

### ISSUE THREAD

**Issue by alanlisten** [pyro-ppl/pyro#1775]
### Issue Description

1. What is the difference between these two pieces of code（model1 and model2）? I feel that the second piece of code has defaulted to the batch_shape is independent, does not require a plate to declare.
2. I also found that inside the model function, if a variable calls the to_event() function, after the trace function is converted, the value of fn is Indepent() instead of the corresponding distribution function e.g. Normal().

### Code Snippet
```
def model1():
    with pyro.plate("d_plate", 3):
          dd = pyro.sample("dd", Normal(torch.zeros(3,4), 1).to_event(1))
```

```
def model2():
    dd = pyro.sample("dd", Normal(torch.zeros(3,4), 1).to_event(1))
```

(For the second question)
```
pyro.clear_param_store()
trace2 = poutine.trace(model2).get_trace()
trace2.nodes["dd"]
```

then will print：
{'type': 'sample',
 'name': 'dd',
 **'fn': Independent(),**   **# Why and What is Independent()? Why is it not normally distributed?**
 'is_observed': False,
 'args': (),
 'kwargs': {},
 'value': tensor([[ 0.3348,  0.5870, -0.6854, -0.6906],
         [-0.3144, -0.2633, -0.0274,  1.6625],
         [-1.4609,  0.0729, -0.4075, -2.3067]]),
 'infer': {},
 'scale': 1.0,
 'mask': None,
 'cond_indep_stack': (),
 'done': True,
 'stop': False,
 'continuation': None}

**Comment by fritzo** [comment]
Hi @alanlisten this is a great question, but if we answer it on github it becomes relatively invisible once the issue is closed. Could you instead ask at https://forum.pyro.ai so others can benefit from an answer? Thanks!
